packages/calling/src/CallingClient/calling/CallerId/types.ts

```diff
@@ -18,31 +18,6 @@ export type PhotoType = {
   value: string;
 };
 
-export type ResourceType = {
```

Comment:
Moved this to common types as it's being used by both calling client and contacts now.